### PR TITLE
feat(webapp): update timeline appearance and refactor flot plugins

### PR DESCRIPTION
**Pull request by pavelpashkovsky**
Short summary on this PR
We don't edit any code in Flot anymore. But there are cases when new functionality based on some built-in plugins. In case of upgrading time selection functionality I extracted `react-flot/flot/jquery.flot.selection.min` into a file `./TimelineChartSelection`, where I customised existing logic in accordance with task requirements. It's important to note, that there are 2 types of timeline selection: `double` and `single`. All customisation was applied only for `single` type of selection. `Double` works in old fashion way (using built-in Flot selection, not plugin-built one).

image

image
- - -
I also extended in a bit TimelineChartWrapper API. Now you can set colors for selection boundaries and overlay. Adding colors depending on color mode is in progress

image

### CODE REVIEW

**Comment by pavelpashkovsky** [comment]
looks like selection boundaries and overlay are too much pale.

**Comment by pavelpashkovsky** [comment]
@Rperry2174, I can be wrong, but I've just remembered that you wanted to remove these captions several months ago? Should I do this?

image

File: webapp/javascript/pages/ContinuousComparisonView.tsx [hunk 1]
```diff
@@ -95,15 +99,26 @@ function ComparisonApp() {
               dispatch(actions.setFromAndUntil({ from, until }));
             }}
             markings={{
-              left: { from: leftFrom, to: leftUntil, color: leftColor },
-              right: { from: rightFrom, to: rightUntil, color: rightColor },
+              left: {
+                from: leftFrom,
+                to: leftUntil,
+                color: leftColor,
+                overlayColor: leftColor.alpha(0.3),
```
Comment (Rperry2174) [inline webapp/javascript/pages/ContinuousComparisonView.tsx:106]: What does `overLayColor` correspond to? is it this 
image

**Comment by Rperry2174** [comment]
> @Rperry2174, I can be wrong, but I've just remembered that you wanted to remove these captions several months ago? Should I do this?

Yeah we can remove these now. 

Given the current way that the code is structured what would be the steps to be able to control the selection like this?
time_selector_mockup_00_3

File: webapp/javascript/pages/ContinuousComparisonView.tsx (hunk 1 above)
Comment (pavelpashkovsky) [reply webapp/javascript/pages/ContinuousComparisonView.tsx:105]: @Rperry2174 you are right, it's visible semi transparent area between selection boundaries, you can set it's color by changing `overlayColor ` prop

**Comment by Rperry2174** [comment]
almost there! I definitely like these selectors better.. Can we make this icon an "open" hand:
image

**Comment by pavelpashkovsky** [comment]
@Rperry2174, I changed resize cursor to open hand, please re/create-server

**Review by Rperry2174 — APPROVED** [review]
LGTM! Nice work @pavelpashkovsky

**Pull request merged**